sophia-vote-contract.md

```diff
@@ -6,11 +6,11 @@ This tutorial takes a look at a smart contract, written in Sophia ML for a votin
 ## Prequisites
 - Installed **docker** (take a look at [this site](https://docs.docker.com/compose/install/) in case you didn't)
 - Installed **aecli** (take a look at [this tutorial](https://github.com/aeternity/tutorials/blob/master/account-creation-in-ae-cli.md#installing-aecli) to remind yourself on installing the javascript version of aecli)
-- Installed **forgae** (take a look at [this section](https://github.com/aeternity/tutorials/blob/master/smart-contract-deployment-in-forgae.md#installing-forgae))
+- Installed **aeproject** (take a look at [this section](https://github.com/aeternity/aepp-aeproject-js))
 
 
 ## Setting up project and development environment 
-First we have to initialize our project where we write our smart contract. In order to do this we are going to use `forgae`.
+First we have to initialize our project where we write our smart contract. In order to do this we are going to use `aeproject init`.
 
 ## Smart contract
 In Sophia ML we have a state which is the place to store data on-chain - and it is the only part in the smart contract that can be mutated (overwritten).
```

Comment:
```suggestion
In Sophia we have a state which is the place to store data on-chain - and it is the only part in the smart contract that can be mutated (overwritten).
```

Reply:
Changes made. Can you resolve the conversation?